### PR TITLE
Garena.Garena version 2.0.1909.2618

### PR DESCRIPTION
**Pull request by RayneWater**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/14547)

### CODE REVIEW

**Comment by RayneWater** [comment]
yeet garena on winget pkgssssss

File: manifests/g/Garena/Garena/2.0.1909.2618/Garena.Garena.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Garena.Garena
+PackageVersion: 2.0.1909.2618
+Installers:
+- Architecture: neutral
```
Comment (ItzLevvie) [inline manifests/g/Garena/Garena/2.0.1909.2618/Garena.Garena.installer.yaml:6]:
```suggestion
- Architecture: x86
```

File: manifests/g/Garena/Garena/2.0.1909.2618/Garena.Garena.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Garena.Garena
+PackageVersion: 2.0.1909.2618
+PackageLocale: en-US
+Publisher: Garena
+PackageName: Garena
+License: Copyright (C) 2010-2019 Garena. All rights reserved.
+ShortDescription: Garena Installer
```
Comment (ItzLevvie) [inline manifests/g/Garena/Garena/2.0.1909.2618/Garena.Garena.locale.en-US.yaml:9]:
```suggestion
ShortDescription: The platform for gamers to play, connect, explore, and more
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Pull request merged**